expression/builtin_cast_vec_test.go

```diff
@@ -57,6 +57,7 @@ var vecBuiltinCastCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETJson, childrenTypes: []types.EvalType{types.ETDecimal}},
 		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETDecimal}},
 		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETInt}},
+		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETString}},
```

Comment:
We need to introduce a new `dataGenerator` for generating a time-format string. The default generator will never generates a string with time-format.